gcs/CONFIGURATION.md

```diff
@@ -248,6 +257,8 @@ When configured, a specified authorization handler will be used to authorize
 Cloud Storage API requests before executing them. The handler will throw
 `AccessDeniedException` for rejected requests if user does not have enough
 permissions (not authorized) to execute these requests.
+Note: Using a `AccessTokenProvider` is the preferred way to perform authentication 
```

Comment:
Why this is the case?

Reply:
Because using the new interface of `StorageAccessTokenProvider` gives both authentication and authorization (i.e. provides the token and checks permissions).